### DISCUSSION TITLE
1 Tesla account, 2 cars?

### DISCUSSION THREAD

**Discussion by sherwinmartin** [teslamate-org/teslamate#2970] [Q&A]
I have 1 Tesla account with 2 cars. Teslamate only detects the most recent car. I am the primary driver of both cars. If I login to my Tesla account they are both there. I have restarted the containers. I have logged out and in with Teslamate. Any ideas?

solved: my model y has been asleep all night in the garage. i guess it has to be awake to register with Teslamate... duh. just got Teslamate running so I'm a bit new.

**Comment by cwanja** [comment]
Do they happen to be named the same? Is there an entry in the database for the other car?

**Reply by sherwinmartin** [reply]
no one is Model X and the other Model Y

**Reply by sherwinmartin** [reply]
nevermind. my model y has been asleep in the garage overnight. i just woke it up and now it shows up.
